sql/parser/eval.go

```diff
@@ -448,6 +448,8 @@ type EvalContext struct {
 	NodeID        uint32
 	StmtTimestamp time.Time
 	TxnTimestamp  time.Time
+	Location      string
```

Comment:
Its a feature very rarely used and I see sql/executor being on the highly
travelled path and I'd hate to slow it down. As it is I hate the fact that
we need to move location info from session to eval context. perhaps we
should add session to evalcontext, and pick up the pieces from there when
needed?

On Mon, Sep 28, 2015 at 12:24 PM Tamir Duberstein <notifications@github.com>
wrote:

> In sql/parser/eval.go
> <https://github.com/cockroachdb/cockroach/pull/2698#discussion_r40572527>:
>
> > @@ -448,6 +448,8 @@ type EvalContext struct {
> >  	NodeID        uint32
> >  	StmtTimestamp time.Time
> >  	TxnTimestamp  time.Time
> > +	Location      string
>
> why not put a time.Location in here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2698/files#r40572527>.
>